finetune/model.py

```diff
@@ -32,6 +33,8 @@ class PredictMode:
     NORMAL = "NORM"
     PROBAS = "PROBA"
     GENERATE_TEXT = "GEN_TEXT"
+    MLM_IDS = "mlm_ids"
```

Comment:
Stylistically kinda strange.  Really doesn't matter but would prefer to keep our caps convention consistent.